### ISSUE TITLE
No IMDb false match

### ISSUE THREAD

**Issue by jonandeva** [L4GSP1KE/Upload-Assistant#280]
Is there a way to force no IMDb? I'm trying to upload a recent show, which does not have an IMDb page. I put in --tmdb tv/123456 and it keeps trying to attach in IMDb for a show of the same name from 1978. I have tried --imdb 0 and --no-imdb, but I guess those don't work.

**Comment by swannie-eire** [comment]
> Is there a way to force no IMDb? I'm trying to upload a recent show, which does not have an IMDb page. I put in --tmdb tv/123456 and it keeps trying to attach in IMDb for a show of the same name from 1978. I have tried --imdb 0 and --no-imdb, but I guess those don't work.

Without more info on what you trying to upload. i would suggest you go to TMDB and make sure that the wrong imdb ID was not added to the TMDB entry as thats more than likely the issue.

**Comment by jonandeva** [comment]
> Without more info on what you trying to upload. i would suggest you go to TMDB and make sure that the wrong imdb ID was not added to the TMDB entry as thats more than likely the issue.

No. There is no IMDb ID attached to it. This is the TMDB https://www.themoviedb.org/tv/229953-chopper-squad. It keeps linking to this one: https://www.imdb.com/title/tt0165003/
